Looks like you added a dependency which you forgot to save in the `devDependencies` array. [Tests are failing on `Error: Cannot find module 'chai-http'`](https://travis-ci.org/Lever-age/api/builds/169732213)

--- a/config.json
+++ b/config.json
@@ -1,0 +1,8 @@
+{
+ "listen": {
+   "port": 8228
+ },
+ "storage": {
+   "path": "test/data/db.sqlite"
+ }
+}

--- a/package.json
+++ b/package.json
@@ -15,6 +15,7 @@
   },
   "devDependencies": {
     "chai": "^3.5.0",
+    "chai-http": "^3.0.0",
     "eslint": "^3.6.1",
     "eslint-config-standard": "^6.2.0",
     "eslint-plugin-promise": "^2.0.1",

